packages/webview_flutter/example/lib/main.dart

```diff
@@ -47,3 +57,52 @@ class SampleMenu extends StatelessWidget {
     );
   }
 }
+
+class NavigationControls extends StatelessWidget {
+  const NavigationControls(this._webViewControllerFuture)
+      : assert(_webViewControllerFuture != null);
+
+  final Future<WebViewController> _webViewControllerFuture;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        FutureBuilder<WebViewController>(
+          builder: (BuildContext context,
+                  AsyncSnapshot<WebViewController> snapshot) =>
+              IconButton(
+                  icon: const Icon(Icons.arrow_back_ios),
+                  onPressed: () async {
+                    final WebViewController controller = snapshot.data;
```

Comment:
This will be null before the future completed, you should probably check if the connectionState is done and that there's no error before using the data.